SafeguardSessions/README.md

```diff
@@ -64,6 +66,12 @@ You might have forgotten to import the CA X.509 certificate of your SPS applianc
 
 ---
 
+C) If you have successfully upgraded your custom connector and tried to fetch data, but receive the following message, revisit the [Upgrading the Power BI Connector] section.
```

Comment:
Can you separate the new error from the others by putting a new separator line before and after the new error scenario?

If you check, each logically different error is separated by a separator line (`---`). It is more eye-catching if you read the .md file in preview mode of your text editor, or on github:
https://github.com/OneIdentity/SafeguardPowerBI/blob/fix/szekersz/SafeguardSessions/README.md#common-errors

Reply:
Okay, I get the logic now. I also removed the C enumeration.